CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/events/handling/MCSaplingGrowTreeEvent.java

```diff
@@ -0,0 +1,55 @@
+/*package crafttweaker.mc1120.events.handling;
```

Comment:
You removed the interface but only commented out the implementation?

Reply:
I meant to delete this.